test-api-generator/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.citrusframework</groupId>
+        <artifactId>citrus</artifactId>
+        <version>4.3.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>citrus-test-api-generator</artifactId>
+    <name>Citrus :: Test API Generator</name>
+    <description>Citrus Test API Generator</description>
+    <packaging>pom</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.openapitools</groupId>
+            <artifactId>openapi-generator</artifactId>
+            <version>7.5.0</version>
```

Comment:
pretty sure these version is always the same as for the maven plugin (https://github.com/citrusframework/citrus/pull/1160/files#diff-a85c5bf4e7da63f6dbd8421e55cf9fe495a2136897f394368514bca2d5daadfeR110)). would probably be worht adding to the global managed versions?